elyra/metadata/schemas/kfp.json

```diff
@@ -47,7 +47,7 @@
           "title": "Kubeflow Pipelines API Endpoint",
           "description": "The Kubeflow Pipelines API endpoint",
           "type": "string",
-          "format": "uri",
+          "pattern": "^((?:https?:\\/\\/)?[^./]+(?:\\.[^./]+)+(?:\\/.*)?)$",
```

Comment:
Security: How do we know that when I leave out https://, Elyra will still use https (and not http)?

Reply:
I have modified the logic to append "https" if the scheme is missing as secured connection is now mandated.

Reply:
lgtm, I guess